src/containers/UserForm.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from "react-redux";
+import { submitUserInfoAction } from "../actions/submitUserInfoAction";
+import { updateUserInfoFormAction } from "../actions/updateUserInfoFormAction";
+import UserForm from "../components/UserForm";
+
+const mapStateToProps = state => ({
+    info: state.userReducer.info || {},
+    form: state.userReducer.form || {},
```

Comment:
is setting these defaults here a thing? Shouldn't the store/reducers handle this instead?

Reply:
Eeh, yeah, hmm, I guess it could be avoided